cfgov/jinja2/v1/ask-cfpb/article-page.html

```diff
@@ -0,0 +1,98 @@
+{% extends 'layout-2-1-bleedbar.html' %}
+{% import 'templates/render_block.html' as render_block with context %}
+{% import 'templates/streamfield-sidefoot.html' as streamfield_sidefoot with context %}
+
+{% block content_modifiers -%}
+    {{ super() }} ask-cfpb-page ask-cfpb-page__article
+{%- endblock %}
+
+{% block content_main %}
+    <div class="eyebrow">{{ _(page.get_category_display()) }}</div>
```

Comment:
Where is `get_category_display()` defined?

Reply:
It's an [extra instance method](https://docs.djangoproject.com/en/dev/ref/models/instances/#django.db.models.Model.get_FOO_display) on the category model because it has `choices` set.

Reply:
Thanks for teaching me that one, @virginiacc !